### PR TITLE
Fix console warnings/errors and prevent multiple popups from multiple FCL messages

### PR DESCRIPTION
**Pull request by huyndo**
- [x] Fix minor console warnings/errors
- [x] Allow only one popup on the extension at a time
- [x] Shows an alert when clicking a command on UI without the previous popup closed
- [x] Sends host tabId to popup instead of querying for the right tab
- [x] Adds a loader to block UI when waiting for a command to be finished

### CODE REVIEW

File: packages/extension/src/Popup.js [hunk 1]
```diff
@@ -32,8 +32,7 @@ function Popup() {
     chrome.tabs &&
       chrome.tabs.query(
         {
-          active: true,
-          currentWindow: false,
+          url: "http://localhost:3000/*"
```
Comment (gregsantos) [inline packages/extension/src/Popup.js]: Need to look into this so we don't rely on hard coded url

Comment (huyndo) [reply packages/extension/src/Popup.js]: I changed this to passing around the host tabId and use that to send messages

File: packages/extension/src/background.js [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+      chrome.scripting.executeScript({
+        target: { tabId: hostTabId },
+        function: () =>
+          alert(
+            "Extension is ignoring given request because it can only have one popup at a time. Make sure to close the current popup before making another request"
+          ),
+      });
+    }
+  } catch (error) {
+    console.log("error: ", error);
```
Comment (gregsantos) [inline packages/extension/src/background.js]: Any reason to throw this? Can the app handle gracefully if there is a problem opening the extension?

File: packages/extension/src/index.js [hunk 3]
```diff
@@ -6,11 +6,16 @@ import Popup from "./Popup";
 
 import "./index.css";
 
+// get tabId of the tab that opened the popup to establish connection with FCL
```
Comment (gregsantos) [inline packages/extension/src/index.js:9]: 🤘🏼

File: packages/extension/src/background.js (hunk 2 above)
Comment (huyndo) [reply packages/extension/src/background.js]: I put that there during debugging and forgot to remove it. It's gone now.

**Review by gregsantos — APPROVED** [review]

**Pull request merged**